R/check_tu_tudtc_across_visit.R

```diff
@@ -10,7 +10,8 @@
 #'
```

Comment:
@sarabodach do you think this check should filter to records with non-missing date?  It seems TU records are often dateless